tests/virt_autotest/login_console.pm

```diff
@@ -121,7 +121,7 @@ sub login_to_console {
         }
     }
 
-    unless (is_tumbleweed or check_screen([qw(grub2 grub1 prague-pxe-menu)], 210)) {
+    unless (is_tumbleweed or check_screen([qw(grub2 grub1 prague-pxe-menu)], get_var('AUTOYAST') && !get_var("GUI_INSTALL") ? 1 : 210)) {
```

Comment:
I think it would be better to lower outer timeout to 180 and increase inner timeout to 120, so
```
unless (is_tumbleweed or check_screen([qw(grub2 grub1 prague-pxe-menu)], get_var('AUTOYAST') && !get_var("GUI_INSTALL") ? 1 : 180)) {
    ..........
    ..........
    check_screen([qw(grub2 grub1 prague-pxe-menu)], 120);
```

Reply:
Please be aware that default TIMEOUT_SCALE is 3, so the real timeout value is the value * 3.

Reply:
I think 180 and 120 is more suitable for general case. There should not be such big difference between outer and inner timeout, namely the first and second try.

Reply:
Agree. I suppose Tony did not do this because it is a separate improvement from autoyast installation. But since it is minor effort, doing it together is also nice choice.  I am fine with either choice.

Reply:
@tonyyuan1 It would be better you can help update this. This is very trivial. Thanks.

Reply:
No problem. Did you test to verify that the two new values work as expected?

Reply:
When you finish all updates/fixes, you can help re-run some verifications together. Should be enough.

Reply:
I want get this ticket over with soon.  can't effort to rerun SOME.  Just one http://10.67.183.130/tests/3824

Maybe I do it with my next PR or you do it with your PR if it's not enough.